pycbc/inference/sampler/base_mcmc.py

```diff
@@ -436,13 +436,27 @@ def set_p0(self, samples_file=None, prior=None):
         self._p0 = p0
         return self.p0
 
-    def set_initial_conditions(self, initial_distribution=None,
-                               samples_file=None):
+    def set_initial_conditions(self, cp):
```

Comment:
Delete this function; you're now doing this in either `resume_from_checkpoint` or `from_config`.